src/content/docs/agents/php-agent/php-agent-api/index.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: PHP agent API
+template: apiIndexPage
+---
+
+The PHP agent API allows you to extend the functionality of the PHP agent. The agent API is included by default with your installation, so no additional configuration is required to use the agent.
+
+Among other things, you can use the API to [customize your app name](/docs/agents/php-agent/php-agent-api/newrelic_set_appname); [collect errors](/docs/agents/php-agent/php-agent-api/newrelic_notice_error); record [custom attributes](/docs/agents/php-agent/php-agent-api/newrelic_add_custom_parameter), [custom events](/docs/agents/php-agent/php-agent-api/newrelic_record_custom_event), and [custom metrics](/docs/agents/php-agent/php-agent-api/newrelic_custom_metric); and [enable](/docs/agents/php-agent/php-agent-api/newrelic_get_browser_timing_header) or [disable](/docs/agents/php-agent/php-agent-api/newrelic_disable_autorum) browser monitoring. For an introduction to this API, see [Guide to PHP agent API](/docs/agents/php-agent/api-guides/guide-using-php-agent-api).
+
+<Callout variant="tip">
+
+Enclose these API calls in a conditional check for the PHP agent so that your code will work even if the agent is not present:
+
+```php
+if (extension_loaded('newrelic')) {
+    newrelic_set_appname($name);
+  }
```

Comment:
[nit] Could you fix the indentation in this code block (here and the `src/manual-edits` copy) to ensure the braces and content are indented appropriately?

Reply:
looks like prettier is forcing that to de-dent. should i overwrite?

Reply:
I mean the contents inside the code block, not the code block itself:

Instead of:

````
```php
if (extension_loaded('newrelic')) {
    newrelic_set_appname($name);
  }
```
````

it should be:

````
```php
if (extension_loaded('newrelic')) {
  newrelic_set_appname($name);
}
```
````

Reply:
indented the contents of the code block itself.

Reply:
yep gotcha. i think should be good now